lib/config/rpcProviderProdConfig.json

```diff
@@ -40,5 +40,11 @@
     "useMultiProviderProb": 0.1,
     "providerInitialWeights": [1, 0, 0, 0],
     "providerUrls": ["INFURA_42161", "QUICKNODE_42161", "NIRVANA_42161", "ALCHEMY_42161"]
+  },
+  {
+    "chainId": 8453,
+    "useMultiProviderProb": 0.01,
+    "providerInitialWeights": [1, 0, 0, 0],
+    "providerUrls": ["QUICKNODE_8453", "INFURA_8453", "ALCHEMY_8453", "NIRVANA_8453"]
```

Comment:
We probably don't want to shadow request infura then, with so many "header not found". It's good we have 4 providers for base.

Reply:
Thanks for the suggestion. I removed Infura from the provider list.